src/widgets/WebexMeeting/WebexMeeting.jsx

```diff
@@ -25,27 +25,38 @@ export default class WebexMeetingWidget extends Component {
     });
     this.state = {
       adapterConnected: false,
+      meeting: null,
     };
     this.adapter = new WebexSDKAdapter(webex);
+    this.meetingSubscription = null;
   }
 
   async componentDidMount() {
     await this.adapter.connect();
     // Once adapter connects, set our app state to ready.
     this.setState({adapterConnected: true});
+
+    this.meetingSubscription = this.adapter.meetingsAdapter
+      .createMeeting(this.props.meetingDestination)
+      .subscribe((meeting) => this.setState({meeting}));
```

Comment:
`createMeeting` returns a [meeting objecthttps://github.com/webex/component-adapter-interfaces/blob/master/src/MeetingsAdapter.js#L20] which doesn't have a `meeting` property

Reply:
actually, the meeting object is not destructured here, it's received as a whole and put on a state property

Reply:
You're right, this is `setState` 🤦🏻‍♀️, no hooks here. Should you make it a shallow copy though? I don't want to run into state issues because React is not picking up the change within the object